framework/py/flwr/common/record/array.py

```diff
@@ -265,19 +265,28 @@ def deflate(self) -> bytes:
         return add_header_to_object_body(object_body=obj_body, cls=self)
 
     @classmethod
-    def inflate(cls, object_content: bytes) -> Array:
+    def inflate(
+        cls, object_content: bytes, children: dict[str, InflatableObject]
+    ) -> Array:
         """Inflate an Array from bytes.
 
         Parameters
         ----------
         object_content : bytes
             The deflated object content of the Array.
 
+        children : dict[str, InflatableObject]
+            Unused. Dict of children InflatableObjects mapped to thier Object ID.
+            These children enable the full inflation of the Array.
+
```

Comment:
Just to avoid potential confusion. We can roll it back once we allow Arrays to have children.

```suggestion
        children : dict[str, InflatableObject]
            Must be an empty dictionary. Arrays do not support child objects.
            Providing any children will raise a ValueError.
```